### DISCUSSION TITLE
Technology

### DISCUSSION THREAD

**Discussion by onyxius** [zencq/NomNom#86] [Q&A] [answered]
First off, thank you so much for putting in the time to create this mod.  I tried joining discord but it just gives me an error unable to join.

Now on to Technology.  I am going through updating or replacing and adding some tech to my multi-tool but there seems to be a lot of missing technology options unless I'm reading or doing something wrong.  Will more be released with the next version and when is the hopeful ETA for the new version?

**Comment by cengelha** [comment, marked as answer]
What error do you get when you try to join on Discord?

The latest version is updated for Outlaws so everything should be there. What exactly are you missing?

There is no ETA for the beta yet. Still too much to do tell.
